corehq/apps/app_manager/templates/app_manager/form_summary_base.html

```diff
@@ -63,7 +63,7 @@ <h5>
                 </ul>
                 <!-- /ko -->
                 <ol data-bind="foreach: questions">
-                    <li data-bind="visible: isVisible, attr: { class: diffClass }">
+                    <li data-bind="visible: isVisible, attr: { class: attributeDiff['label'] ? attributeDiff['label'] : '' }">
```

Comment:
These expressions would be more concise as `attributeDiff['label'] || ''`. Do you know if you just did `attributeDiff['label']`, would knockout insert the string "undefined" as the class?

Reply:
(realized this is outdated, removing comment)